testsuite/utils/constants.py

```
@@ -0,0 +1,121 @@
+# Tracing get_traces retry (fibonacci backoff, 7 attempts).
+TRACING_MAX_RETRIES = 7
+
+# HTTPX request retry (fibonacci backoff, 8 attempts).
+HTTP_BACKOFF_MAX_RETRIES = 8
+
+# --- SpiceDB ---
+
+# HTTP client timeout for SpiceDB API calls (seconds).
+SPICEDB_CONNECTION_TIMEOUT = 30.0
+
+# SpiceDB relationship readiness retry (5s × 3).
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)